### PR TITLE
feat: Prettier

### PR DESCRIPTION
**Pull request by kristianbinau**
## Description
Installed Prettier and ran it : )

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [x] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+node_modules
+pnpm-lock.yaml
+package-lock.json
+prisma
```
Comment (frederikpyt) [inline .prettierignore:4]:
```suggestion
prisma
build
```

**Review by frederikpyt — CHANGES_REQUESTED** [review]
Have you looked at https://mskelton.medium.com/auto-formatting-code-using-prettier-and-github-actions-ed458f58b7df

He seems to have gotten it working

**Comment by kristianbinau** [comment]
> Have you looked at https://mskelton.medium.com/auto-formatting-code-using-prettier-and-github-actions-ed458f58b7df
> 
> He seems to have gotten it working

Okay that's good, you decide if commit message is good haha

**Review by frederikpyt — APPROVED** [review]
Pretty 🥰

**Pull request merged**